templates/drupal8/files/README.md

```diff
@@ -1,43 +1,26 @@
-# Drupal project template for Platform.sh
+# Drupal 8 for Platform.sh
 
-This project provides a starter kit for Drupal 8 projects hosted on [Platform.sh](http://platform.sh). It
-is very closely based on the [Drupal Composer project](https://github.com/drupal-composer/drupal-project).
+This template builds Drupal 8 using the [Drupal Composer project](https://github.com/drupal-composer/drupal-project) for better flexibility.  It also includes configuration to use Redis for caching, although that must be enabled post-install in `.platform.app.yaml`.
 
-## Starting a new project
```

Comment:
Since we anticipate that these will all be created through the UI, I removed the "how to install" instructions since you're already in the midst of installing when you see it.

If someone sees this just on GitHub, they can use as a reference (described below) or go install it themselves.